### ISSUE TITLE
TorrentDay says no configured cookies but I have configured cookies

### ISSUE THREAD

**Issue by ZMOT7S** [pymedusa/Medusa#7596]
**Describe the bug**
Sorry to keep making bug reports. I'm trying to get everything setup since switching to linux a week ago. I am a noob and I apologize. 

My issue is that TorrentDay says 

`020-01-17 19:53:32 WARNING	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] You haven't configured the required cookies. Please login at TorrentDay, and make sure you have copied the following cookies: ('uid', 'pass')`

I have verified and logged in multiple times. I always get the same uid pass. I am using the cookies for torrentday.com/t. See images below. Blacked out the middle but you can see they match. I noticed in the image I'm clicked on / instead of t but they both have the same pass and uid.

**To Reproduce**
Add torrentday as a provider and try to search for something. 

**Expected behavior**
It should search for the file without error. 

**Screenshots**
https://i.imgur.com/8dvdCJw.png

https://i.imgur.com/Ckwtgtx.png

**Medusa (please complete the following information):**

    OS: ubuntu 18.04.3 LTS
    Branch: master
    Commit: 942040a
    Python version: 3.6.9
    Database version: 44.14

**Debug logs (at least 50 lines):**
I know you want at least 50 lines but this is where the issue started. Errors before that are related to qbittorrent not connecting, which I have fixed. If you really want 50 lines, let me know. 

<details>

```
2020-01-17 19:59:16 INFO	SEARCHQUEUE-DAILY-SEARCH :: [942040a] No needed episodes found
2020-01-17 19:59:16 INFO	SEARCHQUEUE-DAILY-SEARCH :: [942040a] Beginning daily search for new episodes
2020-01-17 19:53:32 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] Performing season pack search for Bob's Burgers
2020-01-17 19:53:32 WARNING	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] You haven't configured the required cookies. Please login at TorrentDay, and make sure you have copied the following cookies: ('uid', 'pass')
2020-01-17 19:53:32 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] Performing episode search for Bob's Burgers
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: IPTorrents :: [942040a] Performing episode search for Bob's Burgers
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Using manual search providers
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Building internal name cache for Bob's Burgers
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Finished processing 3136 scene exceptions.
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Updating exception_cache and exception_season_cache
2020-01-17 19:53:31 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Beginning manual search for: Bob's Burgers - S10E11 - Drumforgiven
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] Performing season pack search for Bob's Burgers
2020-01-17 19:46:55 WARNING	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] You haven't configured the required cookies. Please login at TorrentDay, and make sure you have copied the following cookies: ('uid', 'pass')
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: TorrentDay :: [942040a] Performing episode search for Bob's Burgers
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: IPTorrents :: [942040a] Performing episode search for Bob's Burgers
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Using manual search providers
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Building internal name cache for Bob's Burgers
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Finished processing 3136 scene exceptions.
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Updating exception_cache and exception_season_cache
2020-01-17 19:46:55 INFO	FORCEDSEARCHQUEUE-MANUAL-194031 :: [942040a] Beginning manual search for: Bob's Burgers - S10E11 - Drumforgiven
```
</details>

**Additional context**

**Comment by medariox** [comment]
Make sure you are logged in with the same user/pass at TD. Try to restart Medusa. Try a backlog search first.

**Comment by ZMOT7S** [comment]
I did a restart and still the same issue. It is the same user/pass at TD. I've verified multiple times on different browsers. I notice TD has an option for custom url. I tried it blank at first and then with https://www.torrentday.com and https://www.torrentday.com/t but neither solved the issue. 

How can I force a backlog search? When I go to backlog overview, I don't have any missing episodes, so I can't search. 

Also side problem, I added the show "Dave" on FXX. On TVDB it shows the correct show but on medusa it's gives an image for "Dave Hughes Collection." https://www.thetvdb.com/series/dave
